build.gradle

```diff
@@ -1,6 +1,54 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'com.palantir.docker' version '0.21.0'
+    id 'com.palantir.docker-compose' version '0.21.0'
+    id 'com.diffplug.gradle.spotless' version '3.20.0'
+}
+
 group 'multi-int-store'
 version '0.1.0-SNAPSHOT'
 
 repositories {
     mavenCentral()
-}
\ No newline at end of file
+}
+
+allprojects {
+
+    apply plugin: 'com.diffplug.gradle.spotless'
+    apply plugin: 'maven-publish'
+    apply plugin: 'java'
+
+    sourceCompatibility = 1.11
+    targetCompatibility = 1.11
+
+    spotless {
```

Comment:
❗️Add documentation to the README.md about `gradlew spotlessApply`

Reply:
:+1: